Artem_K/hw-1.0-table/css/styles.css

```diff
@@ -0,0 +1,132 @@
+body {
+background-color: grey;
+background-image: url(01.svg), url(01.svg);
+background-size: 95px;
+/*размер картинки*/
+background-repeat: repeat-y, repeat-y;
+background-position: 5% 2%, 95% 2%;
+/*позиция по координатам для левого и правого бекграунда картинки*/
+background-attachment: fixed;
+/* Фиксируем фон веб-страницы */
+}
+
+img {
+  height: 100px;
+}
+
+.divimg {
+  height: 100px;
+  display: inline-block;
+  margin-left: 5%;
+  padding: 10px 10px
+}
+
+h1 {
+  height: 100px;
+  display: inline-block;
+  vertical-align: middle;
+  margin-left: 17%;
+  position: inherit;
+}
+
+.shapka {
+  margin-top: 25px;
+  margin-left: 15%;
+  margin-right: 15%;
+  height: 120px;
+  width: 70%;
+  background-color: white;
+  border: 3px groove silver;
+  border-radius: 30px;
+}
+
+h2 {
+  text-align: center;
+}
+
+h2 span {
+  text-transform: uppercase;
+  text-decoration: underline;
+}
+
+
+/*.table1, td {
+  width: 60%;
+  margin: auto;
+  border: 2px solid #063436;
+  margin-top: 2%;
+  background-color: white;
+}*/
+
+/*.td1 {
+  width: 30%;
+}*/
+.container {
+  width: 70%;
+  margin-left: 15%;
+  margin-right: 15%;
+  /*font-size: 0;*/
+}
+.menu {
+  height: 370px;
+  width: 20%;
+  border: 2px solid black;
+  display: inline-block;
+  background-color: white;
+  box-sizing: border-box;
+}
+
+.main {
+  height: 370px;
+  width: 70%;
+  border: 2px solid black;
+  display: inline-block;
+  background-color: white;
+  padding-left: 30px;
+  box-sizing: border-box;
+}
+
+
+.ul1 {
```

Comment:
try set name for classes that describe for what use this class. 'ul1' - not inform name